### PR TITLE
Update names and labels with correct capitalization, etc.

### LINKED ISSUE #1466: Lowercase "and" in default WOF and English names
[404429267](https://spelunker.whosonfirst.org/id/404429267/) `Grovesend And Waungron Community` should be **Grovesend and Waungron Community** with lowercase `and`.

image

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst-data/whosonfirst-data/issues/1466.

This PR updates `name:` properties, `wof:name` properties, and `wof:label` properties.

- Corrects incorrect capitalization in `wof:name` and `wof:label` properties
  - " And " -> " and"
  - " Of " -> "of"
  - " With " -> " with"
  - " In " -> " in"
  - " On " -> " on"
  - " The " -> " the"
- Backfills `name:eng_x_preferred` when not present, but `name:eng_x_variant` is present
- Saves incorrectly capitalized forms of names in the `name:eng_x_variant` property, following lifecycle rules so we're not dropping name values.
- Adds "Saint" variants when "St. " string is present in a name

No PIP work required, can merge once approved.

### CODE REVIEW

File: data/112/600/518/5/1126005185.geojson [hunk 1]
```diff
@@ -21,7 +21,10 @@
         "Newton-with-Scales"
     ],
     "name:eng_x_preferred":[
-        "Newton-with-Scales"
+        "Newton with Scales"
+    ],
+    "name:eng_x_variant":[
+        "Newton With Scales"
```
Comment (nvkelso) [inline data/112/600/518/5/1126005185.geojson]: Please add the - variant here, too: 

- `Newton-with-Scales`

File: data/136/075/304/1/1360753041.geojson [hunk 2]
```diff
@@ -15,6 +15,9 @@
     "mz:hierarchy_label":1,
     "mz:is_current":-1,
     "name:eng_x_preferred":[
+        "St. Georges and Priorslee"
+    ],
+    "name:eng_x_variant":[
         "St. Georges And Priorslee"
```
Comment (nvkelso) [inline data/136/075/304/1/1360753041.geojson]: Please add St. spelled out as variant

**Review by nvkelso — APPROVED** [review]
One fix for the `-` that we should do for life cycle rules... the `saint` thing looks more widespread so file generic followup issue, please.

File: data/136/075/299/7/1360752997.geojson [hunk 3]
```diff
@@ -15,7 +15,11 @@
     "mz:hierarchy_label":1,
     "mz:is_current":-1,
     "name:eng_x_preferred":[
-        "Thirkleby High And Low With Osgodby"
+        "Thirkleby High and Low with Osgodby"
+    ],
+    "name:eng_x_variant":[
+        "Thirkleby High And Low With Osgodby",
+        "Thirkleby High and Low with Osgodby"
```
Comment (stepps00) [inline data/136/075/299/7/1360752997.geojson]: I'll also take care of duplicates...

File: data/136/075/304/1/1360753041.geojson (hunk 2 above)
Comment (stepps00) [reply data/136/075/304/1/1360753041.geojson]: Adding this in a new commit, along with some changes to label properties. There are inconsistencies I've noticed in label and name values, I'll clean them up in a new commit.

**Comment by stepps00** [comment]
@nvkelso - the two mot recent commits address comments and take care of a few additional issues in the name and label properties.

**Review by nvkelso — APPROVED** [review]

**Pull request merged**